ghcide/src/Development/IDE/Types/Monitoring.hs

```diff
@@ -0,0 +1,30 @@
+module Development.IDE.Types.Monitoring
+(Monitoring(..)
+) where
+
+import           Data.Int
+import           Data.Text                (Text)
+
+-- | An abstraction for runtime monitoring.
```

Comment:
could you say a little about what a "gauge" and a "counter" are? I think I know but it would be nice to write it down.

Reply:
These are EKG names, I'll add a link